modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc

```diff
@@ -26,6 +26,8 @@ When you setup new clients which should be managed, take care that you put them
 [NOTE]
 ====
 If you want to manage systems in an already existing network, you must configure **peering**.
+
+For more information, see link:https://learn.microsoft.com/en-us/azure/virtual-network/tutorial-connect-virtual-networks?tabs=portal#create-virtual-network-peer[this tutorial].
```

Comment:
Instead of "this tutorial" I'd write "Microsoft Azure documentation" or "Azure documentation".